scripts/telemetry/telemetry.cfg

```diff
@@ -0,0 +1,5 @@
+[ufm-telemetry-endpoint]
+host = 10.209.36.68
```

Comment:
1. please make the default is 127.0.0.1
2. rename the cfg file to be consistent with other scripts and make it telemetry.sample.cfg